### DISCUSSION TITLE
中文吹水吧

### DISCUSSION THREAD

**Discussion by theagora** [agorahub/_meta#8] [General]
### 答疑 • 分享 • 互動

**Comment by andyof** [comment]
噏㗕喇

**Comment by theagora** [comment]
## 無權力者的權力 —— Gear噏
![gu1](https://images.weserv.nl/?url=na.cx/i/T7B6ogq.jpg)

| [01](https://images.weserv.nl?url=na.cx/i/T7B6ogq.jpg) | [02](https://images.weserv.nl?url=na.cx/i/Zt7dNH4.jpg) | [03](https://images.weserv.nl?url=na.cx/i/MD3wzn5.jpg) | [04](https://images.weserv.nl?url=na.cx/i/Q3Arogq.jpg) | [05](https://images.weserv.nl?url=na.cx/i/Evic3gp.jpg) |
| -- | -- | -- | -- | -- |
| [06](https://images.weserv.nl?url=na.cx/i/YQE7R9K.jpg) | [07](https://images.weserv.nl?url=na.cx/i/JCtsR9L.jpg) | [08](https://images.weserv.nl?url=na.cx/i/4ArRzn4.jpg) | [09](https://images.weserv.nl?url=na.cx/i/znqZJgp.jpg) | [10](https://images.weserv.nl?url=na.cx/i/s3PWLuK.jpg) |

**Comment by theagora** [comment]
## 2020香港大事回顧 —— 幽光
![2020](https://images.weserv.nl/?url=i.redd.it/p0lrqzi1yig61.jpg)

| [01](https://images.weserv.nl?url=i.redd.it/ex6k3wj2yig61.jpg) | [02](https://images.weserv.nl?url=i.redd.it/sdix0o63yig61.jpg) | [03](https://images.weserv.nl?url=i.redd.it/lfttywq3yig61.jpg) | [04](https://images.weserv.nl?url=i.redd.it/cmfimwd4yig61.jpg) | [05](https://images.weserv.nl?url=i.redd.it/4mj2j065yig61.jpg) | [06](https://images.weserv.nl?url=i.redd.it/5zejbnq5yig61.jpg) |
| -- | -- | -- | -- | -- | -- |
| [07](https://images.weserv.nl?url=i.redd.it/17i8nxd6yig61.jpg) | [08](https://images.weserv.nl?url=i.redd.it/y57xsg17yig61.jpg) | [09](https://images.weserv.nl?url=i.redd.it/mpm55wk7yig61.jpg) | [10](https://images.weserv.nl?url=i.redd.it/ev50qkb8yig61.jpg) |  [11](https://images.weserv.nl?url=i.redd.it/o3v2wjy8yig61.jpg) | [12](https://images.weserv.nl?url=i.redd.it/5gb7ocm9yig61.jpg) |

**Comment by theagora** [comment]
## 香港已死？ —— Gear噏
![gu1](https://images.weserv.nl/?url=i.redd.it/sw4692t519k61.jpg)

| [01](https://images.weserv.nl?url=i.redd.it/sw4692t519k61.jpg) | [02](https://images.weserv.nl?url=i.redd.it/ry281ga619k61.jpg) | [03](https://images.weserv.nl?url=i.redd.it/hixryqq619k61.jpg) | [04](https://images.weserv.nl?url=i.redd.it/4fjz524719k61.jpg) | [05](https://images.weserv.nl?url=i.redd.it/blrzedl719k61.jpg) |
| -- | -- | -- | -- | -- |
| [06](https://images.weserv.nl?url=i.redd.it/yaphqsx719k61.jpg) | [07](https://images.weserv.nl?url=i.redd.it/qdpm7bn819k61.jpg) | [08](https://images.weserv.nl?url=i.redd.it/ca7bsh2919k61.jpg) | [09](https://images.weserv.nl?url=i.redd.it/9ult1y3a19k61.jpg) | [10](https://images.weserv.nl?url=i.redd.it/lw0z91ja19k61.jpg) |

**Comment by theagora** [comment]
## 都再見唔到喇！

| ![01](https://images.weserv.nl?url=i.redd.it/6lt5h1jzq8771.jpg) | ![02](https://images.weserv.nl?url=i.redd.it/xr0d91jzq8771.jpg) |
| -- | -- |
| ![03](https://images.weserv.nl?url=i.redd.it/k1gkwzizq8771.jpg) | ![04](https://images.weserv.nl?url=i.redd.it/58php1jzq8771.jpg) |

**Comment by theagora** [comment]
## 見字讀書 裝備腦袋
阿塗 | 2021-07-14 續 #6 登山讀書會
> 《美麗新世界》是英國作家赫胥黎於一九三二年發表的作品，與《一九八四》、《我們》並列為西方三大反烏托邦科幻小說，也是一本指向未來，卻點出人類文明發展困頓的科幻經典。在香港這個荒謬大時代重讀起來，或許更有一番切身體會。
> 在美麗新世界的國度裡，不再有悲哀、哭號、疼痛，也不再有反對、異議、抗爭。為了安穩，人人都各安本份，飾演被指派的階級、角色，按著既定程序，做出「正確」的事情。所有慾望都不需壓抑，遇有不愉快的心情，便依靠政府提供的藥物梭麻來解憂，你要甚麼，政府統統都能給你！這就是美麗的新世界。

![Cp170](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018082-cae532a7-fc64-475c-8bc4-6bf2668cee4f.jpg)

| ![Cp171](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018160-b51f200d-acd9-4511-9753-1e2c90372e76.jpg) | ![Cp172](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018190-6fb9c3aa-fbea-446a-ae92-999fbfc36dfb.jpg) | ![Cp173](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018212-b0317a00-39f2-44ee-ab54-26de400f08a3.jpg) |
| -- | -- | -- |
| ![Cp174](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018234-8d80e46b-4a01-4ac4-9654-ded958a6c2f2.jpg) | ![Cp175](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018245-a3693f76-cff7-4a07-b2e8-575da2f04db7.jpg) | ![Cp176](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018250-cf89c470-14a9-4855-b3c0-d3fd90e75be4.jpg) |
| ![Cp177](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018259-ee3c370f-89ed-48dd-be52-76417edc06f6.jpg) | ![Cp178](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018328-8264df47-4903-4c8a-9bfb-2ac801115ff7.jpg) | ![Cp179](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148018337-07deb16c-1208-40ae-b9b2-349488108437.jpg) |

**Comment by theagora** [comment]
## 若為自由故
阿塗 | 2021-07-15 續 #6 登山讀書會
> 自由為何物？若人民追求的安穩社會，同時要人民放棄部分自由去換取，我們該如何取捨 ? 事無大小都有政府貼心關顧，不須再煩惱地選擇，一切娛樂慾望都能被滿足，這就能沒有憂愁嗎？這是這篇漫畫的小叩問，亦是經典小說《美麗新世界》中的大哉問。
> 《美麗新世界》是英國作家赫胥黎於一九三二年發表的作品，與《一九八四》、《我們》並列為西方三大反烏托邦科幻小說，也是一本指向未來，卻點出人類文明發展困頓的科幻經典。在香港這個荒謬大時代重讀起來，或許更有一番切身體會。



| ![Cp181](https://user-images.githubusercontent.com/59945368/148019741-2ebdedaa-b3e1-45e8-a534-f36ed3de418f.jpg) | ![Cp182](https://user-images.githubusercontent.com/59945368/148019758-10b9b12b-db23-4608-941b-6247ea19c021.jpg) | ![Cp183](https://user-images.githubusercontent.com/59945368/148019775-90a6ad54-ae63-455f-8fba-8967d5856270.jpg) |
| -- | -- | -- |
| ![Cp184](https://user-images.githubusercontent.com/59945368/148019853-d519767a-c633-4063-84b8-605a42e2efe3.jpg) | ![Cp185](https://user-images.githubusercontent.com/59945368/148019866-38f86179-f75c-49d0-a5b5-9cb89e645d06.jpg) | ![Cp186](https://user-images.githubusercontent.com/59945368/148019872-fdbe0b84-73c0-4e49-86e9-9347b47b1966.jpg) |
| ![Cp187](https://user-images.githubusercontent.com/59945368/148019882-b7a86cfd-f25b-4530-ad6c-07492df40408.jpg) | ![Cp188](https://user-images.githubusercontent.com/59945368/148019888-53364f71-0437-4b61-be9f-2362d5a02999.jpg) | ![Cp189](https://user-images.githubusercontent.com/59945368/148019900-78aa1a81-54db-45b6-9b5a-783a928fd1d2.jpg) |

**Comment by theagora** [comment]
## 悲慘之世 焰猶不絕 黑夜縱黑 終會破曉
阿塗 | 2021-11-07 續 #6 登山讀書會
> “For the wretched of the earth, 
there is a flame that never dies. 
Even the darkest night will end 
and the sun will rise.” 

Les Misérables

|  | ![Cp191](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148022101-d78f6cb8-cd14-4112-9f10-1308909fb828.jpg) | ![Cp190](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148022073-86bf8de7-370b-42ea-b20c-f20ee862e5a1.jpg) |
| -- | -- | -- |
| ![Cp192](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148022133-dda068f9-8c0d-424d-b30f-2da223e7bf1b.jpg) | ![Cp193](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148022153-e1d116a3-cfa9-4dbb-bc02-806cabd1ed8b.jpg) | ![Cp194](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148022160-78f79bca-23d8-446c-b5f4-f0115d26cc70.jpg) |

![Cp199](https://images.weserv.nl?url=user-images.githubusercontent.com/59945368/148035042-b245adf4-35e2-4076-9be5-4349362a7581.jpg)

**Reply by theagora** [reply]
### Production Samples
LLHK團隊 | 2021-08-18 #6 登山讀書會
| ![CRC200](https://user-images.githubusercontent.com/59945368/148029731-bd9d25ce-5f70-44d8-86de-0db338825a1b.png) | ![CRC201](https://user-images.githubusercontent.com/59945368/148029749-96323c5b-1ddd-4a5f-8963-4ce067017e6c.png) |
| -- | -- |
| ![CRC202](https://user-images.githubusercontent.com/59945368/148029765-5c281e01-a68f-4b48-9738-19e3d2ef06c0.png) | ![CRC203](https://user-images.githubusercontent.com/59945368/148029777-7853a719-2dab-4391-95d6-d5f6ad1a5351.png) |

**Comment by theagora** [comment]
## 编程随想（阮晓寰）的判决书全文
2023.03

| ![p1](https://images.weserv.nl?url=i.redd.it/2n6uonpxxapa1.jpg) | ![p2](https://images.weserv.nl?url=i.redd.it/sf0tgnpxxapa1.jpg) |
| -- | -- |
| ![p3](https://images.weserv.nl?url=i.redd.it/s1qsxnpxxapa1.jpg) | ![p4](https://images.weserv.nl?url=i.redd.it/kc5hfppxxapa1.jpg) |
| ![p5](https://images.weserv.nl?url=i.redd.it/ab4zpopxxapa1.jpg) | ![p6](https://images.weserv.nl?url=i.redd.it/9riuxopxxapa1.jpg) |
